e2e/test/scenarios/metabot/metabot.cy.spec.ts

```diff
@@ -40,11 +39,11 @@ describe("Metabot UI", () => {
     });
   });
 
-  describeEE("EE", () => {
+  // token feature has been disabled so skipping for now - there's currently decent coverage for UI related things in our unit tests
+  describe.skip("EE", () => {
```

Comment:
The `metabot_v3` flag appears to have been removed from the cypress all features token. All of the UI tests are currently also covered in unit tests, so I think it's fine to temp disable these.